### DISCUSSION TITLE
Remove default_hour and default_minute from config?

### DISCUSSION THREAD

**Discussion by micahellison** [jrnl-org/jrnl#1465] [General]
Currently, we have two key values in the config that add an hour and minute to a new entry when you specify a date but no time. Here they are with their defaults:
```
default_hour: 9
default_minute: 0
```

Here's how they work -- note how the 09:00 was added to the timestamp:
```
>jrnl yesterday: This happened yesterday
[Entry added to default journal]

>jrnl -on yesterday
2022-04-22 09:00 This happened yesterday
```

However, we aren't consistent in how we use this. For instance, when I run `jrnl --edit` and change the bracketed timestamp from `[2022-04-22 09:00]` to `[yesterday]`, jrnl saves the entry's timestamp as `[2022-04-22 00:00]`.

It's also not used in search terms, which makes sense: if I'm searching `-from yesterday`, I want to see all entries from yesterday, not all entries from yesterday after 9:00am.

I'm wondering how much people actually care about this behavior. It could simplify things to remove the keys entirely and just let entries with no times default to midnight. Does anyone have any feedback on this?

**Comment by DSiekmeier** [comment]
This might be related: in some other applications I saw the option to set "all day" for an entry. This is all the stored (in addition to the timestamp) and used for viewing purposes.
